.circleci/tests.sh

```diff
@@ -92,4 +92,30 @@ done
 
 echo ""
 echo "Found trace \"$TRACE_ID\"."
+
+echo "Checking for broken links"
+
```

Comment:
deserves a large comment about why these patterns are used including 123456-123456 (or substitute with variable)

Reply:
Yeah, `123456-123456` sounds very arbitrary. Could we reuse the obtained trace_id?

Reply:
yeah! we can.

Reply:
@JBAhire changed it for TRACE_ID